interface-definitions/interfaces_wireguard.xml.in

```diff
@@ -98,9 +111,14 @@
                     <format>ipv6</format>
                     <description>IPv6 address of remote tunnel endpoint</description>
                   </valueHelp>
+                  <valueHelp>
```

Comment:
We might re-thing the CLI node here is it would probably be a better idea to add a new `hostname` CLI node for this particular dyndns case and make it mutually exclusive to `address`